frontend/src/main/java/de/iav/frontend/service/IdService.java

```diff
@@ -0,0 +1,24 @@
+package de.iav.frontend.service;
+
+
+import java.security.SecureRandom;
+
+public class IdService {
```

Comment:
Wofür benötigst du im FE einen IdService?
Alle IDs sollen typischerweise im BE generiert werden.

Je nach Anwendungsfall kann das ja sonst vom Client der in den Händen des Endanwenders ist, missbraucht werden